src/components/offerPage/index.jsx

```diff
@@ -0,0 +1,403 @@
+                                {
+                                    required: true,
+                                    message: offersErrorMessages.EMPTY_SETTLEMENT_MESSAGE,
+                                },
+                            ]}
+                        >
+                            <Input name="region" type="hidden" placeholder="Enter your region"/>
+                        </Form.Item>
+
+                        <Form.Item
+                            name="goodCategory"
+                            label="Select good categorie: "
+                            labelAlign="left"
+                            rules={[
+                                {
+                                    required: true,
+                                    message: offersErrorMessages.EMPTY_GOOD_CATEGORY_MESSAGE,
+                                },
+                            ]}
+                        >
+                            <Select placeholder="Select good categories" className="select-good-categorie">
+                                {data?.map((res, idx) => (
+                                    <Option value={res.name} key={idx}>
+                                        {res.name.toLowerCase()}
+                                    </Option>
+                                ))}
+                            </Select>
+                        </Form.Item>
+
+                        <Form.Item
+                            name="dates"
+                            label="Select date: "
+                            labelAlign="left"
+                            rules={[
+                                {
+                                    required: true,
+                                    message: offersErrorMessages.EMPTY_START_DATE_MESSAGE,
+                                },
+                            ]}
+                        >
+                            <RangePicker
+                                disabledDate={disabledDate}
+                                disabledTime={disabledRangeTime}
+                                showTime={{
+                                    hideDisabledOptions: true,
+                                    defaultValue: [
+                                        moment("00:00:00", "HH:mm"),
+                                        moment("12:00:00", "HH:mm"),
+                                    ],
+                                }}
+                                format="YYYY-MM-DD HH:mm"
+
+                            />
+                        </Form.Item>
+
+                        <Form.Item
+                            name="goodsWeight"
+                            label="Select good weight: "
+                            labelAlign="left"
+                            rules={[
+                                {
+                                    pattern: "^[0-9]",
+                                    message: offersErrorMessages.NOT_VALID_GOOD_WEIGHT_MESSAGE,
```

Comment:
An incomprehensible message is displayed to the user

![image](https://user-images.githubusercontent.com/56841893/172664285-d52405f1-97a2-45b7-bd88-58dd3d1b83e7.png)

Reply:
![image](https://user-images.githubusercontent.com/56602775/172701657-8aee0ca4-c8a4-4475-9581-f3c296b89525.png)